extensions/extensions.json

```diff
@@ -16,6 +16,7 @@
   "utilities",
   "sound",
   "iframe",
+  "CubesterYT/KeySimulation",
```

Comment:
What reason do you have for putting this extension here? It feels rather high up for such a niche functionality.

Reply:
It's a utility type extension, so I put it with other utilities.